problem_builder/dashboard.py

```diff
@@ -238,10 +238,26 @@ class DashboardBlock(StudioEditableXBlockMixin, XBlock):
         help=_("Toggles if numeric values are displayed"),
         scope=Scope.content
     )
+    header_html = String(
+        display_name=_("Header HTML"),
+        default="",
+        help=_("Additional HTML to include after the heading."),
```

Comment:
This XBlock is a summary report that summarises all answers a student has given.  The report can be viewed in the LMS, and can also be downloaded, which is why I said "included" rather then "displayed".

I tried again, but English not being my first language, I'd appreciate feedback on my attempt. :)